lib/propolis/src/hw/virtio/viona.rs

```diff
@@ -70,11 +71,16 @@ enum VRingState {
 
 struct Inner {
     poller: Option<PollerHdl>,
+    iop_state: Option<u16>,
     vring_state: [VRingState; 2],
 }
 impl Inner {
     fn new() -> Self {
-        Self { poller: None, vring_state: [Default::default(); 2] }
+        Self {
+            poller: None,
+            iop_state: None,
+            vring_state: [Default::default(); 2],
```

Comment:
take it or leave it: this is identical to what a derived `Default` implementation would do...